ngui/ui/src/components/DataSourceCredentialFields/AwsAssumedRoleInputs/AwsAssumedRoleInputs.tsx

```diff
@@ -7,19 +7,25 @@ import AwsUseAwsEdpDiscount from "../AwsUseAwsEdpDiscount";
 
 const AwsAssumedRoleInputs = ({
   readOnlyFields = [],
-  showAssumedRoleCredentialsInModal = false
+  showAssumedRoleCredentialsInModal = false,
+  showAdvancesOptions = true
```

Comment:
I suppose there is a typo here; should it be `showAdvancedOptions`?


```suggestion
  showAdvancedOptions = true
```

Reply:
done

Reply:
done